mirror.go

```diff
@@ -22,10 +22,12 @@ const (
 	dockerHub = "hub.docker.com"
 	quay      = "quay.io"
 	gcr       = "gcr.io"
+	k8s_gcr   = "k8s.gcr.io"
```

Comment:
Can you please follow the golang standard and use camel case for this variable, more or less how `dockerHub` is not `docker_hub` :) Or just call it `k8s` as this is the defactory `k8s` repo.

Reply:
updated, thanks for the comment.